play-json/jvm/src/main/scala/play/api/libs/json/JsonConfig.scala

```diff
@@ -186,9 +193,13 @@ object JsonConfig {
   private[json] def loadPreserveZeroDecimal: Boolean =
     prop(preserveZeroDecimalProperty, defaultPreserveZeroDecimal)(_.toBoolean)
 
-  // Default settings, which can be controlled with system properties.
-  // To override, call JacksonJson.setConfig()
-  val settings: JsonConfig =
+  /** Override configuration */
+  def setConfig(jsonConfig: JsonConfig): Unit = JacksonJson.initConfig(jsonConfig)
+
+  /** Current configuration. */
+  def settings: JsonConfig = JacksonJson.get.jsonConfig
```

Comment:
This should probably be called `get` or `getConfig`, but I'm leaving it for backwards compatibility.